docs/tutorials/python/data_augmentation.md

```diff
@@ -49,13 +47,21 @@ One of the most convenient ways to augment your image data is via arguments of [
 We show a simple example of this below, after creating an `images.lst` file used by the [`ImageIter`](https://mxnet.incubator.apache.org/api/python/image/image.html?highlight=imageiter#mxnet.image.ImageIter). Use [`tools/im2rec.py`](https://github.com/apache/incubator-mxnet/blob/master/tools/im2rec.py) to create the `images.lst` if you don't already have this for your data.
 
 ```python
-!echo -e "0\t0.000000\timages/0.jpg" > ./data/images.lst
+path_to_image = os.path.join("images","0.jpg")
+index = 0
+label = "0.000000"
+list_file_content = "{}\t{}\t{}".format(index, label, path_to_image)
```

Comment:
Could use {:5f}

Reply:
I find it slightly less visual that way but will update